qiita_pet/templates/edit_study.html

```diff
@@ -20,168 +20,71 @@
 </style>
 
 <script>
-// qutocomplete combobox code modified from http://jqueryui.com/autocomplete/#combobox, retrieved 8 Sept 2014
-(function( $ ) {
-    $.widget( "custom.combobox", {
-      _create: function() {
-        this.wrapper = $( "<span>" )
-          .addClass( "custom-combobox" )
-          .insertAfter( this.element );
-
-        this.element.hide();
-        this._createAutocomplete();
-        this._createShowAllButton();
-      },
-
-      _createAutocomplete: function() {
-        var selected = this.element.children( ":selected" ),
-          value = selected.val() ? selected.text() : "";
-
-        this.input = $( "<input>" )
-          .appendTo( this.wrapper )
-          .val( value )
-          .attr( "title", "" )
-          .addClass( "custom-combobox-input ui-widget ui-widget-content ui-state-default ui-corner-left" )
-          .autocomplete({
-            delay: 0,
-            minLength: 0,
-            source: $.proxy( this, "_source" )
-          })
-          .tooltip({
-            tooltipClass: "ui-state-highlight"
-          });
-
-        this._on( this.input, {
-          autocompleteselect: function( event, ui ) {
-            ui.item.option.selected = true;
-            this._trigger( "select", event, {
-              item: ui.item.option
-            });
-          },
-
-          autocompletechange: "_removeIfInvalid"
+$(document).ready(function() {
+    $("#new_person_div").hide();
+     $( "#{{creation_form.principal_investigator.id}}" ).chosen({
+        allow_single_deselect: true,
```

Comment:
I think we should try avoiding last elements with `,s` as they behave differently in different browsers. For example, the length of this could be 1 or 2 but without the comma it will always be 1. Note that if this change was done when it was first reported, this will had been already merged!!